src/bundle/DependencyInjection/SchebTwoFactorExtension.php

```diff
@@ -36,16 +36,31 @@ public function load(array $configs, ContainerBuilder $container): void
         }
 
         // Load two-factor modules
-        if (isset($config['email']['enabled']) && true === $config['email']['enabled']) {
-            $this->configureEmailAuthenticationProvider($container, $config);
+        if (isset($config['email']['enabled'])) {
+            $config['email']['enabled'] = (bool) $container->resolveEnvPlaceholders($config['email']['enabled'], true);
+
+            if (true === $config['email']['enabled']) {
```

Comment:
I'd suggest to change this to

```php
if (isset($config['email']['enabled']) && $this->resolveFeatureFlag($config['email']['enabled'])) {
```

and add a `resolveFeatureFlag` method for the env-transformation and boolean casting logic. The name `resolveFeatureFlag` can be discussed, was just the first thing that came to my mind.